src/widget/wwaveformviewer.h

```diff
@@ -49,8 +49,6 @@ private slots:
         m_waveformWidget = nullptr;
     }
 
-    void slotWaveformUpdated();
```

Comment:
The thing is that only GLSL renderer was suffering from the issue of not updating the waveform (that is the issue from #841 I couldn't solve then).